test/integration/select_test.lua

```diff
@@ -258,6 +258,20 @@ pgroup.test_negative_first = function(g)
     local objects = crud.unflatten_rows(result.rows, result.metadata)
     t.assert_equals(objects, helpers.get_objects_by_idxs(customers, {6}))
 
+    -- id <= 6
```

Comment:
But it's `id <= 2`

Reply:
Yes, my mistake. There was another same problem, fixed.